languageApp/src/screens/Login/Login.tsx

```diff
@@ -67,8 +67,11 @@ export const Login: React.FC = ({navigation}: NavigationType) => {
           dispatch(setAuthState('checked'));
           dispatch(setEmail(result.email));
           dispatch(setName(result.firstName));
+          // dispatch(setBeginner_progress(result.beginner_progress))
+          // dispatch(setIntermediate_progress(result.intermediate_progress))
+          // dispatch(setAdvance_progress(result.advance_progress))
         }
-      } catch (error) {}
+      } catch (error) {console.error(error)}
     },
```

Comment:
You could opt for a SnackBar to show errors instead of printing them on console ;)